pandas/core/strings.py

```diff
@@ -633,21 +633,21 @@ def _str_extract_frame(arr, pat, flags=0):
 
 def str_extract(arr, pat, flags=0, expand=True):
     r"""
+    Return the match object corresponding to regex `pat`.
+
     For each subject string in the Series, extract groups from the
-    first match of regular expression pat.
+    first match of regular expression `pat`.
 
     Parameters
     ----------
     pat : string
-        Regular expression pattern with capturing groups
+        Regular expression pattern with capturing groups.
     flags : int, default 0 (no flags)
-        re module flags, e.g. re.IGNORECASE
-
+        Re module flags, e.g. re.IGNORECASE.
```

Comment:
In this case it is fine to not capitalize the first word, as it refers to a python module. Therefore, would you actually quote it like ``` ``re`` ``` ? (and the same for `re.IGNORECASE`)